styles.css

```diff
@@ -631,14 +646,108 @@ h6 {
 }
 
 .form button {
-  display: flex;
+  /* display: flex; */
   justify-content: center;
   align-items: center;
   text-align: center;
-  color: #6070ff;
+
+  /* color: #6070ff; */
 }
 
 .get-in-touch {
   display: flex;
   flex-direction: row;
 }
+
+@media only screen and (min-width: 768px) {
+  .Template-1 {
+    position: absolute;
+    width: 100%;
+  }
+
+  .header-background-mobile {
+    background-image: url(./media/Header\ bg@2x.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .Toolbar img {
+    display: none;
+  }
+
+  .Toolbar ul {
+    display: flex;
+    list-style-type: none;
+    color: #344563;
+  }
+
+  .Card-Works {
+    flex-direction: row;
+    height: 496px;
+  }
+
+  .Snapshoot-Portfolio img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .Left-Block {
+    width: 544px;
+    height: 445px;
+  }
+
+  .About-myself {
```

Comment:
- [ ] Good job on your design 👍🏽 . Kindly ensure that your about-me section title matches the Figma design. Please increase font-size and font-weight 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/199583677-9777dd78-2ecd-4d41-b8e5-e6e468fae1a7.png)  | ![image](https://user-images.githubusercontent.com/63473584/199583510-13e9873d-159e-41fc-9cc9-a29f1781a131.png)  |